bindings/matrix-sdk-ffi/src/client_builder.rs

```diff
@@ -648,6 +669,15 @@ impl ClientBuilder {
     }
 }
 
+#[derive(Clone)]
+/// The store paths the client will use when built.
+struct SessionPaths {
+    /// The path that the client will use to store its data.
+    data_path: String,
+    /// The path that the client will use to store its caches.
+    cache_path: Option<String>,
```

Comment:
Why is it optional? If that's necessary, can you update the doc to clarify please?

Reply:
It probably isn't necessary tbf. I just followed the situation in the main SDK, but it would be cleaner in the FFI to force users to set this.